src/checkoutservice/README.md

```diff
@@ -39,6 +39,6 @@ dep ensure --vendor-only
   Transformation](https://opentelemetry.io/docs/collector/deployment/#agent)
 - :construction: [Push
   Metrics](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#push-metric-exporter)
-- :construction:[SLO Metrics](https://github.com/openslo/openslo#slo): :construction:
+- :construction:[SLO Metrics](https://github.com/openslo/openslo#slo)
```

Comment:
Another minor fix 😄:

```suggestion
- :construction: [SLO Metrics](https://github.com/openslo/openslo#slo)
```

Reply:
There goes my linux contrib rights lol. Updated in all files